xmtp_proto/src/gen/xmtp.keystore_api.v1.rs

```diff
@@ -428,9 +515,42 @@ impl ErrorCode {
         }
     }
 }
+/// JobType is used to specify the type of job the caller would like info on
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord, ::prost::Enumeration)]
+#[repr(i32)]
+pub enum JobType {
+    Unspecified = 0,
+    RefreshV1 = 1,
+    RefreshV2 = 2,
+    RefreshPppp = 3,
+}
+impl JobType {
+    /// String value of the enum field names used in the ProtoBuf definition.
+    ///
+    /// The values are not transformed in any way and thus are considered stable
+    /// (if the ProtoBuf definition does not change) and safe for programmatic use.
+    pub fn as_str_name(&self) -> &'static str {
+        match self {
+            JobType::Unspecified => "JOB_TYPE_UNSPECIFIED",
+            JobType::RefreshV1 => "JOB_TYPE_REFRESH_V1",
+            JobType::RefreshV2 => "JOB_TYPE_REFRESH_V2",
+            JobType::RefreshPppp => "JOB_TYPE_REFRESH_PPPP",
```

Comment:
Probably a change that needs to happen in the `proto` repo if we care to keep the naming consistent across everything?. cc @rygine pppp -> user_preferences

Reply:
That one only gets persisted locally, so not the end of the world if it changes post-launch. Only side effect is that it'll force a refresh of the topic from the beginning.

Reply:
@neekolas is the PR that needs an update? https://github.com/xmtp/proto/pull/116

Reply:
We can also change that to User Preferences if we wanted to, but that's truly invisible to the end user and can be changed at any time. 

[This was the one I was thinking about](https://github.com/xmtp/proto/pull/114)